tests/shutdown/shutdown.pm

```diff
@@ -3,8 +3,9 @@ use base "openQAcoretest";
 use testapi;
 
 sub run {
-    wait_screen_change { send_key 'ctrl-alt-f3' };
-    type_string "poweroff\n";
+    send_key('ctrl-alt-f3', wait_screen_change => 1) ;
```

Comment:
Is `wait_screen_change => 1` required if we're using an `assert_screen` in the next line anyways?

Reply:
No, shouldn't be required. That was my point